app/calculation/index.js

```diff
@@ -1,22 +1,37 @@
-const buildResponse = (calculationPayload) => {
-  calculate(calculationPayload, 'soilProtection', 6)
-  calculate(calculationPayload, 'permanentGrasslandProtection', 6)
-  calculate(calculationPayload, 'moorlandGrazing', 6)
-  calculate(calculationPayload, 'livestockWelfare', 6)
-  return calculationPayload
-}
+const filterStandardsSummary = require('../standards/filter')
+const standardsCodes = require('../standards/standards-codes')
+const paymentRateCodes = require('./payment-rate-codes')
+const { getStandardsSummary } = require('../api/crown-hosting')
 
-const calculate = (calculationPayload, standardType, amount) => {
-  const standard = calculationPayload.calculation.standards[standardType].actions
-  for (const [key, value] of Object.entries(standard)) {
-    standard[key] = value.map(action => {
-      action.expression = 'x*y'
-      action.value = 616800
-      action.expression = `AREA*${amount}`
-      action.value = action.area * amount
-      return action
+const getParcelPaymentValues = (standardsSummary, standardsCode) => {
+  const actions = filterStandardsSummary(standardsSummary, standardsCode)
+  return actions.filter(p => p.claimAtLevel === 'PARCEL')
+    .map(a => {
+      return {
+        ambitionLevel: paymentRateCodes.find(x => x.code === a.code).ambitionLevel,
+        paymentRate: a.paymentRate
+      }
     })
+}
+
+const calculatePaymentRates = async (callerId, applicationId, parcels, code) => {
+  const standardsSummary = await getStandardsSummary(callerId, applicationId)
```

Comment:
I suppose this is an area where we could use a cached result as it will have already been done when requesting the standards. Not suggesting changing now though